daliuge-engine/dlg/apps/pyfunc.py

```diff
@@ -667,17 +669,21 @@ def _ports2args(self, pargsDict, keyargsDict) -> dict:
                 key = list(inport.keys())[0]
                 inputs_dict[key] = {"name": inport[key], "path": None, "drop":
                     self._inputs[key]}
+            skip_on_input = True
```

Comment:
I don't believe this is necessary any more, as we don't use it in the `identify_named_ports` function.